Specs/Data/Cesium3DTiles/Metadata/MultipleContentsWithMetadata/tileset_1.0.json

```diff
@@ -0,0 +1,105 @@
+{
+  "asset": {
+    "version": "1.0"
+  },
+  "properties": {
+    "id": {
+      "minimum": 0,
+      "maximum": 9
+    },
+    "Longitude": {
+      "minimum": -1.3196959060375169,
+      "maximum": -1.3196607462778132
+    },
+    "Latitude": {
+      "minimum": 0.6988590050687061,
+      "maximum": 0.6988864387845588
+    },
+    "Height": {
+      "minimum": 6.1022464875131845,
+      "maximum": 20
+    }
+  },
+  "extensionsRequired": [
+    "3DTILES_multiple_contents"
+  ],
+  "extensionsUsed": [
+    "3DTILES_multiple_contents",
+    "3DTILES_metadata"
+  ],
+  "extensions": {
+    "3DTILES_metadata": {
+      "schema": {
+        "classes": {
+          "batched": {
+            "properties": {
+              "highlightColor": {
+                "type": "VEC3",
+                "componentType": "UINT8"
+              },
+              "author": {
+                "type": "STRING"
+              }
+            }
+          },
+          "instanced": {
+            "properties": {
+              "numberOfInstances": {
+                "type": "SCALAR",
+                "componentType": "UINT8"
+              },
+              "author": {
+                "type": "STRING"
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "geometricError": 70,
+  "root": {
+    "refine": "ADD",
+    "boundingVolume": {
+      "region": [
+        -1.3197004795898053,
+        0.6988582109,
+        -1.3196595204101946,
+        0.6988897891,
+        0,
+        30
+      ]
+    },
+    "geometricError": 0,
+    "extensions": {
+      "3DTILES_multiple_contents": {
+        "content": [
```

Comment:
@j9liu just to check, for 1.0 + extensions do we prefer "contents" now even in `3DTILES_multiple_contents`?

Reply:
yes, I'll change this